.github/workflows/ci.yaml

```diff
@@ -507,6 +507,58 @@ jobs:
 
       - uses: codecov/codecov-action@v1
 
+  snapshot-release:
```

Comment:
discussion/issue: I'm not a huge fan of releasing every PR's every commit as image. It really pollutes the Docker Hub. Merging on main is fine, but do we really need these for every PR commit? Especially since this will fail the PR build when a user uses a fork (which would be the default for most users).

If you just want the built image available as part of the PR, then you could use the upload-action so it would be available as download.

Reply:
I agree that poluting Dcker Hub is no good. I've got my image out and the related k8ssandra-operator PR is passing. I'm ok with dropping this entirely (pushed a commit doing so)